src/amber/cli/templates/app/README.md.ecr

```diff
@@ -1,53 +1,44 @@
 # <%= @name %>
 
-This project is powered by [Amber Framework](https://amberframework.org/).
+[![Amber Framework](https://img.shields.io/badge/using-amberframework-orange.svg)](https://amberframework.org)
 
-## Installation
+This is a project written using [Amber](https://amberframework.org). Enjoy!
 
-1. [Install required dependencies](https://github.com/amberframework/online-docs/blob/master/getting-started/quickstart/zero-to-deploy.md#install-crystal-and-amber)
-2. Run `shards install`
+## Getting Started
 
-## Usage
+These instructions will get you a copy of this project up and running on your local machine for development and testing purposes. See [deployment](https://amberframework.gitbook.io/amber/deployment) for notes on how to deploy the project on a live system.
```

Comment:
I would break into the new line beginning with "See."
Also, I would say "Please see" and "for more notes on deploying the project in production".